### DISCUSSION TITLE
Welcome to Codewars Discussions!

### DISCUSSION THREAD

**Discussion by kazk** [codewars/codewars.com#2338] [General]
Welcome 👋

We’re trying out GitHub Discussions as a place to connect with other members of our community. We hope that you:

* Ask [questions and answer](https://github.com/codewars/codewars.com/discussions/categories/q-a) them.
* Share [how Codewars has helped](https://github.com/codewars/codewars.com/discussions/categories/kind-words) you.
* Share [ideas](https://github.com/codewars/codewars.com/discussions/categories/ideas).
* Welcome others and are open-minded.

You can also join our Discord.

[![Discord chat](https://img.shields.io/discord/846624424199061524.svg?logo=discord&style=flat)](https://discord.gg/mSwJWRvkHA)

**Comment by MuhammadIsmail-khan** [comment]
i was working on a problem of integer to binary conversion which was showing error and then i run that program on intlj idea which was working properly i dont know what was reason

**Reply by DonaldKellett** [reply]
Feel free to drop by our Discord server for hints on solving Kata: https://discord.com/invite/mSwJWRvkHA

**Comment by Pdzoc** [comment]
Is it allowed to discuss here about politics? I don't set new thread and feel free to delete this comment if Codewars don't want any things like this, but I am curious that you (I mean Codewars) didn't change logo's colors or didn't add note with info "we are united with people from Ukraine" or something similar.

**Reply by Blind4Basics** [reply]
> Is it allowed to discuss here about politics?

Directly here or on CW: no. That's not the point of CW.
If you _really_ want to talk politics with other users, you can take your chance on the discord channel/#random.

But, clearly, if you want to talk about politics, find a place/channel where it's the main purpose. I personally don't see the point of tackling this kind of topic on a __coding__ website/discord...

Cheers
